test/e2e/lua/dynamic_configuration.go

```diff
@@ -111,6 +111,55 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 			Expect(restOfLogs).ToNot(ContainSubstring("first sync of Nginx configuration"))
 		})
 
+		It("configuration module should read from temp file when request body > client_body_buffer_size", func() {
+			resp, _, errs := gorequest.New().
+				Get(fmt.Sprintf("%s?id=endpoints_only_changes", f.NginxHTTPURL)).
+				Set("Host", "foo.com").
+				End()
+			Expect(len(errs)).Should(BeNumerically("==", 0))
+			Expect(resp.StatusCode).Should(Equal(http.StatusOK))
+
+			// Update client-body-buffer-size to 1 byte
+			cm, getErr := f.KubeClientSet.CoreV1().ConfigMaps("ingress-nginx").Get("nginx-configuration", metav1.GetOptions{})
+			Expect(getErr).ToNot(HaveOccurred())
+			oldClientBodyBufferSize, oldClientBodyBufferSizeExists := cm.Data["client-body-buffer-size"];
+
+			cm.Data["client-body-buffer-size"] = "1"
+			_, updateErr := f.KubeClientSet.CoreV1().ConfigMaps("ingress-nginx").Update(cm)
+			Expect(updateErr).ToNot(HaveOccurred())
+
+			replicas := 4
```

Comment:
1 replica is not enough to be over 1 byte?

Reply:
But 2 replicas does look like it is enough :)